packages/navigation-menu/src/lib/navigation-menu/services/menu.service.ts

```diff
@@ -0,0 +1,174 @@
+import { Injectable, TemplateRef, Inject } from '@angular/core';
+import { Observable, Subject, merge, fromEvent } from 'rxjs';
+import { share, tap, filter, mapTo, startWith, map, shareReplay, first, scan, takeUntil, mergeMap } from 'rxjs/operators';
+import { Menu } from '../interfaces';
+
+/**
+ * Singleton helper service to orchestrate the navigation menu.
+ * To not have a dependency on state management frameworks such as redux,
+ * this service holds the menu state and exposes it using an observable.
+ *
+ * Caution should be taken: do not instantiate more than 1 instane of this service,
+ * since it contains subscriptions.
+ */
+@Injectable()
+export class MenuService {
+	// Const variable used to know whether the app is rendered on a mobile screen or not
+	private static readonly MOBILE_VIEWPORT = 576;
```

Comment:
Is there a way to read this variable from one of the branding packages? This one is not correct btw.

Reply:
Our core branding is not prepared to export variables to a webpack configuration. Also, could you then provide me with the correct value?

Reply:
Breakpoints are listed in the Core Branding. I believe the best breakpoint to switch navigation menu mode is 62rem (`$screen-md`).

Reply:
Yes, but these sass variabels can't be read by webpack and processed as js constants, unless we alter the core branding to export these variables.